connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorHeartbeatConnector.java

```diff
@@ -85,6 +89,26 @@ public String version() {
         return AppInfoParser.getVersion();
     }
 
+    @Override
+    public boolean alterOffsets(Map<String, String> config, Map<Map<String, ?>, Map<String, ?>> offsets) {
```

Comment:
Same question as the `MirrorCheckpointConnector` case - why do we want to allow users to modify the offsets?

Reply:
Yeah I've not looked at the background but it looks I agree with Yash, it looks strange